tf-modules/archive/api.tf

```diff
@@ -35,6 +35,7 @@ locals {
       cmr_username                     = var.cmr_username
       databaseCredentialSecretArn      = var.rds_user_access_secret_arn
       dbHeartBeat                      = var.rds_connection_heartbeat
+      DeadLetterProcessingLambda       = aws_lambda_function.process_dead_letter_archive.arn
```

Comment:
Nit.... and not really your PR.  It'd be nice if we had some consistency in the env var casing.

Reply:
..I don't disagree, but out of scope.. 😅